### PR TITLE
getting-started-libvirt: mention how to get coreos-installer

### PR DESCRIPTION
**Pull request by jlebon**
This is the first mention of coreos-installer when reading from the
Getting Started guide. Let's provide more info re. where they can get
coreos-installer from.

Closes: #192

### CODE REVIEW

**Comment by travier** [comment]
I would recommend linking to https://coreos.github.io/coreos-installer/getting-started/ instead as it has all the options for running coreos-installer in a single page.

**Comment by jlebon** [comment]
> I would recommend linking to coreos.github.io/coreos-installer/getting-started instead

Yeah, I considered that, but I'm hesitant to have the "Getting Started" guide link to other pages where there's yet more information that needs to be consumed and understood.

OTOH, I agree just saying "can be run as a container" might not be enough to go on. Though the upstream docs container example is for installing, not downloading. Maybe let's add an example there for that and then we can link directly to it from the FCOS docs?

**Comment by travier** [comment]
We have that in the tutorial too: https://docs.fedoraproject.org/en-US/fedora-coreos/tutorial-setup/
Maybe we should make it a separated included "page"?

**Comment by travier** [comment]
Or maybe we could create a specific "How to download FCOS" section with all examples.

**Comment by jlebon** [comment]
Maybe let's just get this in for now? I agree it could be improved, though at least it's an improvement on what we currently have. :)

**Comment by bgilbert** [comment]
Let's at least provide an example of running from a container, since there are a couple non-obvious options.

```
STREAM="stable"
podman run --pull=always --rm -v $HOME/.local/share/libvirt/images/:/data -w /data \
    quay.io/coreos/coreos-installer:release  download -s "${STREAM}" -p qemu -f qcow2.xz --decompress
```

**Comment by jlebon** [comment]
OK, done!

File: modules/ROOT/pages/getting-started-libvirt.adoc [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+. Fetch the latest image suitable for the `qemu` platform using `coreos-installer` (or https://getfedora.org/coreos/download/[download and verify] it from the web). You can use `coreos-installer` https://quay.io/repository/coreos/coreos-installer[as a container], or on Fedora install it from the repos.
 +
 [source, bash]
 ----
 STREAM="stable"
+# as an installed binary:
 coreos-installer download -s "${STREAM}" -p qemu -f qcow2.xz --decompress -C ~/.local/share/libvirt/images/
+# or as a container:
+podman run --pull=always --rm -v $HOME/.local/share/libvirt/images/:/data -w /data \
+    quay.io/coreos/coreos-installer:release  download -s "${STREAM}" -p qemu -f qcow2.xz --decompress
```
Comment (bgilbert) [inline modules/ROOT/pages/getting-started-libvirt.adoc:12]: Extra space after `release`, duly copypasted from my example.  :disappointed:

**Review by bgilbert — APPROVED** [review]

**Pull request merged**